ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Compat.hs

```diff
@@ -115,6 +115,9 @@ singleEraCompatQuery epochSize slotLen f = go
     goHardFork :: QueryHardFork '[era] result -> m result
     goHardFork GetInterpreter = return $ Qry.mkInterpreter summary
     goHardFork GetCurrentEra  = return $ eraIndexZero
+    goHardFork GetLedgerCfg = undefined
+    goHardFork GetConsensusCfg = undefined
+    -- TODO ^^^ Pattern match(es) are non-exhaustive
```

Comment:
TODO

Reply:
I'm not entirely sure what to do here. If I understand correctly, we'll need to take the ledger and consensus config as new parameters to `singleEraCompatQuery`. Does that sound right?